src/analyses/cfg_dominators.h

```diff
@@ -219,6 +221,7 @@ template <typename T, typename node_indext>
 const node_indext
   dominatorst<T, node_indext>::npos = std::numeric_limits<node_indext>::max();
 
+/// Dominators for each node
```

Comment:
Maybe something like "Compute dominators for each instruction in a GOTO program"?